README.md

```diff
@@ -30,7 +30,7 @@ To use Toolkit in your project:
 
 ## Requirements
 * ArcGIS Maps SDK for Swift
-* The [System Requirements](https://developers.arcgis.com/swift/system-requirements/system-requirements-for-200-6/) are the same as those for the ArcGIS Maps SDK for Swift
+* The [System Requirements](https://developers.arcgis.com/swift/system-requirements/system-requirements-for-200-8/) are the same as those for the ArcGIS Maps SDK for Swift
```

Comment:
This should always link to the latest SDK.
```suggestion
* The [System Requirements](https://developers.arcgis.com/swift/system-requirements) are the same as those for the ArcGIS Maps SDK for Swift
```

Reply:
Nice!